src/components/pages/devices/devices.js

```diff
@@ -63,11 +79,16 @@ class DevicesPage extends Component {
           {this.state.contextBtns}
           <SimControlCenter />
           <ManageFilterBtn />
-          <PcsBtn svg={AddSvg} 
-                  onClick={this.props.openProvisionFlyout} 
+          <PcsBtn svg={AddSvg}
+                  onClick={this.props.openProvisionFlyout}
                   value={lang.PROVISIONDEVICES} />
         </ContextFilters>
-        <PageContent>
+        <PageContent className="devices-grid-container">
+        <div className="timerange-selection">
+          <span className="last-refreshed-text"> {`${lang.LAST_REFRESHED} | `} </span>
```

Comment:
There spaces after and before the `><`, is that intended to add whitespace to the page? If not please remove the spaces.